src/main/java/internet/store/dao/ProductDaoImpl.java

```diff
@@ -0,0 +1,44 @@
+package internet.store.dao;
```

Comment:
```suggestion
package internet.store.dao.impl;
```
move all implementations to impl package